compute: fix typo in metric name

teskje opened the pull request:
`mz_compute_comamnd_history_size` -> `mz_compute_command_history_size`

### Motivation

  * This PR fixes a previously unreported bug.

### Checklist

- [ ] This PR has adequate test coverage / QA involvement has been duly considered.
- [ ] This PR evolves [an existing `$T ⇔ Proto$T` mapping](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/command-and-response-binary-encoding.md) (possibly in a backwards-incompatible way) and therefore is tagged with a `T-proto` label.
- [ ] If this PR will require changes to cloud orchestration, there is a companion cloud PR to account for those changes that is tagged with the release-blocker label ([example](https://github.com/MaterializeInc/cloud/pull/5021)).
  <!-- Ask in #team-cloud on Slack if you need help preparing the cloud PR. -->
- [x] This PR includes the following [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note):
  - N/A

aalexandrov reviewed (approved):
This might also entail some changes in Grafana. It will be good to list dashboard panels that need to be fixed once this is deployed. @umanwizard / @lluki probably are best positioned to source this information .

teskje commented:
I know of https://g-2531aff0fb.grafana-workspace.us-east-1.amazonaws.com/d/grEP2lOVk/compute-monitoring that will need to be adjusted, but there might be more. I'll ask in the cloud channel how to best handle this.

vmarcos commented:
> I know of https://g-2531aff0fb.grafana-workspace.us-east-1.amazonaws.com/d/grEP2lOVk/compute-monitoring that will need to be adjusted, but there might be more. I'll ask in the cloud channel how to best handle this.

This is the only one that I am aware of as well. I agree that it is still a good idea to ask in the cloud channel, since folks may have used this metric in other dashboards.

lluki commented:
I have another one in my playground, but no need to hold it back. I'm checking if we can somehow dump the grafana dashboards to grep through it...

teskje commented:
I've updated @vmarcos' dashboard by adding a second query for the typo-fixed metric. That way it will work with both old and new environments until we have updated all of them. We can do the same to other affected dashboards we might find.

teskje commented:
TFTRs!

lluki commented:
BTW I ran this [script](https://gist.github.com/crisidev/bd52bdcc7f029be2f295?permalink_comment_id=4443773#gistcomment-4443773) to fetch all JSON definitions of our dashboards and grepped through it. Yes it's only used in `vmarcosplayground` and `lukasplayground` 8-) 

I just plugged in `headers="Cookie: grafana_session=XXXXXX"` as authentication that i fetched from my browser session.
